src/utils/webrtc/webrtc.js

```diff
@@ -669,15 +666,15 @@ export default function initWebRTC(signaling, _callParticipantCollection) {
 				checkPeerMedia(peer, video, 'video').then(function() {
 					clearInterval(peer.check_video_interval)
 					peer.check_video_interval = null
-				})
+				}).catch()
```

Comment:
may I suggest `.catch(console.error.bind(this))`? Then it will still log the error

Reply:
The error is already logged inside checkPeerMedia

Reply:
Still, the `then` callback could throw as well …

Reply:
but then we can also jsut get rid of the catch, right?
Because if clearInterval throws....

Reply:
Depends. Some browsers will still show you the error then. Others will just swallow it. Or complain that you're not handling the error.

IIRC `.catch(console.error.log(this))` is the easiest way to "handle" it when you don't care or think it's unlikely to ever get triggered.

Reply:
See my remark as a hint for a "best practice". You're free to ignore it :)